src/main/java/org/wso2/carbon/connector/util/FileConstants.java

```diff
@@ -69,6 +69,8 @@
     public static final String DEFAULT_RESPONSE_FILE = "/response.xml";
     public static final String INCLUDE_PARENT_DIRECTORY = "includeParentDirectory";
     public static final String DEFAULT_INCLUDE_PARENT_DIRECTORY = "false";
+    public static final String INCLUDE_SUBDIRECTORIES = "includeSubdirectories";
+    public static final String DEFAULT_INCLUDE_SUBDIRECTORIES = "true";
```

Comment:
Shall we make this a boolean since it is a constant so that we don't have to parse and take the value?